### PR TITLE
Strip symbols of inferno binaries to decrease size by 24x

### PR DESCRIPTION
**Pull request by marcospb19**
After running `cargo install --path .` to install all `inferno` binaries, here's what I noticed:

```sh
 ~ du -hc .cargo/bin/inferno-*
28M .cargo/bin/inferno-collapse-dtrace
25M .cargo/bin/inferno-collapse-ghcprof
30M .cargo/bin/inferno-collapse-guess
29M .cargo/bin/inferno-collapse-perf
27M .cargo/bin/inferno-collapse-recursive
25M .cargo/bin/inferno-collapse-sample
25M .cargo/bin/inferno-collapse-vsprof
25M .cargo/bin/inferno-collapse-vtune
24M .cargo/bin/inferno-diff-folded
28M .cargo/bin/inferno-flamegraph
263M    total
```

Files were extremely large due to this `debug` value:
```toml
[profile.release]
debug = true
```

Here are the values after stripping (with `strip` binary or key in manifest file):
```sh
 ~ du -hc .cargo/bin/inferno-*
1.2M    .cargo/bin/inferno-collapse-dtrace
1.1M    .cargo/bin/inferno-collapse-ghcprof
1.4M    .cargo/bin/inferno-collapse-guess
1.2M    .cargo/bin/inferno-collapse-perf
1.1M    .cargo/bin/inferno-collapse-recursive
1012K   .cargo/bin/inferno-collapse-sample
1000K   .cargo/bin/inferno-collapse-vsprof
1020K   .cargo/bin/inferno-collapse-vtune
932K    .cargo/bin/inferno-diff-folded
1.4M    .cargo/bin/inferno-flamegraph
11M total
```

And here's what you get by not using `debug` or `strip`:

```sh
 ~ du -hc .cargo/bin/inferno-*
5.6M    .cargo/bin/inferno-collapse-dtrace
5.4M    .cargo/bin/inferno-collapse-ghcprof
5.8M    .cargo/bin/inferno-collapse-guess
5.6M    .cargo/bin/inferno-collapse-perf
5.5M    .cargo/bin/inferno-collapse-recursive
5.4M    .cargo/bin/inferno-collapse-sample
5.3M    .cargo/bin/inferno-collapse-vsprof
5.4M    .cargo/bin/inferno-collapse-vtune
5.3M    .cargo/bin/inferno-diff-folded
5.8M    .cargo/bin/inferno-flamegraph
55M total
```

## TLDR

| debug (current) | none | strip |
|---|---|---|
|263M|55M|11M|

So I added `strip = true`

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -15,6 +15,10 @@ license = "CDDL-1.0"
 
 exclude = ["/tests/**", "/flamegraph/**", "/*.perf"]
 
+[profile.release]
+strip = true   # To use flamegraph on inferno binaries, comment this line
+# debug = true # and uncomment this line.
+
```
Comment (marcospb19) [inline Cargo.toml:21]: I moved this to make it the second section, so that people don't miss it, and added these comments just in case someone tries to use these binaries with `flamegraph`.

**Review by jonhoo — APPROVED** [review]
Great spot, thank you! Would you mind also updating `CHANGELOG.md` and I'll make a release?

**Comment by marcospb19** [comment]
Of course :) done.

**Review by jonhoo — APPROVED** [review]
Thanks!

**Comment by jonhoo** [comment]
Released in 0.11.19 :tada:
